extensions/resteasy-reactive/rest-client-reactive/deployment/src/test/java/io/quarkus/rest/client/reactive/timeout/BuilderReadTimeoutTest.java

```diff
@@ -55,4 +63,36 @@ void shouldNotTimeoutOnDefaultTimeout() {
 
         assertThat(client.slow()).isEqualTo("slow-response");
     }
+
+    @Produces(MediaType.TEXT_PLAIN)
+    @Consumes(MediaType.TEXT_PLAIN)
+    public static interface Client {
```

Comment:
This and the following class where just moved inside the test, nothing else changed